src/components/Table/TableHead/TableHead.jsx

```diff
@@ -128,6 +133,41 @@ const StyledCustomTableHeader = styled(TableHeader)`
   }
 `;
 
+const getColumnDragBounds = (direction, colWidths, index) => {
+  const minColumnWidth = 50;
+  return {
+    direction,
+    minColumnWidth,
+    rightBound: colWidths[index],
+    leftBound: colWidths[index + 1] - minColumnWidth,
+  };
+};
+
+const updateColumnResizeWrapper = (validDrag, wrapperElement) => {
+  const invalidClass = 'column-resize-wrapper--invalid';
+  if (validDrag === false) {
+    wrapperElement.classList.add(invalidClass);
```

Comment:
direct DOM manipulation like this seems like a bad idea, instead we should set an invalid state and Declaratively render the classname based on the state

Reply:
I agree in general, but this class is added to the columnResizer element while it is being dragged and during that DOM action there is no react rendering going on.

Reply:
Sorry. Now I understand. I'll add the validDrag to the state to render while dragging. Will fix it first thing tomorrow.

Reply:
fixed